src/globals/scss/styles.scss

```diff
@@ -62,6 +63,7 @@ $css--reset: false !default;
 @import '../../components/progress-indicator/progress-indicator';
 @import '../../components/breadcrumb/breadcrumb';
 @import '../../components/toolbar/toolbar';
+@import '../../components/time-picker/time-picker';
```

Comment:
@marijohannessen Did you really want this line? Seems that this is for time picker and I'm getting a build error complaining that this SCSS file is missing.